ui/src/modules/app_builder/index.tsx

```diff
@@ -278,10 +279,28 @@ const BuilderMenu: React.FC<{
     importApp(config)
   }
 
+  const sanitize = (values: { [k: string]: any }) => {
+    if (!values) {
+      return
+    }
+
+    const isSerect = values.name === SECRET_NAME
```

Comment:
typo: isSecret
the same in the next line

Reply:
nice catch. thanks!